portfolio/src/main/webapp/index.html

```diff
@@ -61,7 +107,13 @@
         const quote = textFromResponse[quoteIndex];
         helloContainer.innerHTML = quote;
         }
-
+    function submitForm() {
+   
+        const form = document.getElementById('form-reset');
+        form.submit(); // Submit the form
+        form.reset();  // Reset all form data
+        //return false; // Prevent page refresh
```

Comment:
is this commented out code used in final product? If not, please delete. General rule of thumb is delete all debugging/trial codes that you don't end up using in final product

Reply:
I've removed it. Thanks!